test/testcases_QuantumESPRESSO/silicon_LDA/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+cd "$(dirname "$0")"
+PREFIX="silicon_LDA"
+
```

Comment:
Yeah, I'd really rather come up with a more robust way of handling the pseudos though because non-permalink URLs all over the code and testcases is really not ideal.

Reply:
I agree ...